src/cpp/src/tokenizer.cpp

```diff
@@ -91,11 +96,25 @@ class Tokenizer::TokenizerImpl {
         // Try to read tokenizer_config if some token ids or token str are not defined.
         read_tokenizer_config_if_necessary(tokenizer_path); 
 
+        std::map<std::string, ov::Any> plugin_config{{"PERFORMANCE_HINT", "THROUGHPUT"}};
         auto device = "CPU"; // currently openvino_tokenizer supports only CPU
-        m_tokenizer_request = core.compile_model(tokenizer_path / "openvino_tokenizer.xml",
-                                                device).create_infer_request();
-        m_detokenizer_request = core.compile_model(tokenizer_path / "openvino_detokenizer.xml", 
-                                                   device).create_infer_request();
+    
+        m_tokenizer = core.compile_model(tokenizer_path / "openvino_tokenizer.xml",
+                                                device, plugin_config);
+        m_detokenizer = core.compile_model(tokenizer_path / "openvino_detokenizer.xml", 
+                                                   device, plugin_config);
+
+        const size_t INFER_REQUEST_QUEUE_SIZE = 256;
```

Comment:
I suggest to read the compile_model property NUM_STREAMS and set the queue size to such value

Reply:
Good idea. There's optimal_number_of_infer_requests. Read it instead. It's meta plugin aware

Reply:
Done

Reply:
You still read `num_streams`: `const size_t INFER_REQUEST_QUEUE_SIZE = m_tokenizer.get_property(ov::num_streams);`

Reply:
Right, I have changed it to `optimal_number_of_infer_requests` now